hazelcast/src/test/java/com/hazelcast/internal/config/override/ExternalMemberConfigurationOverrideEnvTest.java

```diff
@@ -142,6 +144,22 @@ public void shouldHandleUserCodeDeploymentConfig() throws Exception {
         assertEquals(UserCodeDeploymentConfig.ClassCacheMode.OFF, config.getUserCodeDeploymentConfig().getClassCacheMode());
     }
 
+    @Test
+    public void shouldHandleSplitBrainProtectionConfig() throws Exception {
+        Config config = new Config();
+        config.getSplitBrainProtectionConfig("foo")
+          .setEnabled(true)
+          .setProtectOn(SplitBrainProtectionOn.READ);
+
+        withEnvironmentVariable("HZ_SPLITBRAINPROTECTION_FOO_ENABLED", "true")
+          .and("HZ_SPLITBRAINPROTECTION_FOO_FUNCTIONCLASSNAME", "com.foo.SomeClass")
+          .execute(() -> new ExternalConfigurationOverride().overwriteMemberConfig(config));
+
+        assertTrue(config.getSplitBrainProtectionConfig("foo").isEnabled());
+        assertSame(SplitBrainProtectionOn.READ, config.getSplitBrainProtectionConfig("foo").getProtectOn());
+        assertEquals("com.foo.SomeClass", config.getSplitBrainProtectionConfig("foo").getFunctionClassName());
+    }
+
```

Comment:
I think we need a test for a non-existent config as well. We rely on getting a non-null config for `config.getSplitBrainProtectionConfig(name)` in `MemberDomConfigProcessor`, but it's not tested.

Reply:
There are no non-existent configs. They get instantiated on get (just like our data structures).

Notice that this test actually fetches a _non-existing_ "foo" config at the beginning

Reply:
Ignore this comment, it is tested by the existing config builder tests.